packages/compartment-mapper/src/zip/format-reader.js

```diff
@@ -28,7 +28,17 @@
  *  comment: string, // presumed UTF-8
  * }} CentralDirectoryLocator
  *
- * @typedef {import('./buffer-reader.js').BufferReader} BufferReader
```

Comment:
Oh damn. I didn't see this one coming. Unpleasant but I have no suggested improvement.

Reply:
It’s technically an improvement for the type here to express what’s needed since it doesn’t depend on the concrete type. But, in the future, we will need other work-arounds.